src/components/deploymentConfig/DeploymentTemplateView.tsx

```diff
@@ -0,0 +1,831 @@
+                fetchingReadMe={fetchingReadMe}
+                openReadMe={openReadMe}
+                isReadMeAvailable={isReadMeAvailable}
+                handleReadMeClick={handleReadMeClick}
+            />
+        </div>
+    )
+}
+
+const formatOptionLabel = (option: DeploymentChartOptionType): JSX.Element => {
+    return (
+        <div className="flex left column">
+            <span className="w-100 ellipsis-right">
+                {option.label}&nbsp;{option.version && `(${option.version})`}
+            </span>
+        </div>
+    )
+}
+
+const customValueContainer = (props: any): JSX.Element => {
+    return (
+        <components.ValueContainer {...props}>
+            {props.selectProps.value?.label}&nbsp;
+            {props.selectProps.value?.version && `(${props.selectProps.value.version})`}
+            {React.cloneElement(props.children[1], {
+                style: { position: 'absolute' },
+            })}
+        </components.ValueContainer>
+    )
+}
+
+const CompareWithDropdown = ({ isEnvOverride, environments, selectedOption, setSelectedOption }) => {
+    const [groupedOptions, setGroupedOptions] = useState<DeploymentChartGroupOptionType[]>([
+        {
+            label: '',
+            options: [],
+        },
+    ])
+    const baseTemplateOption = {
+        id: -1,
+        value: '',
+        label: 'Base deployment template',
+    }
+
+    useEffect(() => {
+        if (isEnvOverride) {
+            setGroupedOptions([
+                {
+                    label: '',
+                    options: [baseTemplateOption],
+                },
+                {
+                    label: 'values on other environments',
+                    options: environments.length > 0 ? environments : [{ label: 'No options', value: 0, info: '' }],
+                },
+            ])
+
+            if (!selectedOption) {
+                setSelectedOption(baseTemplateOption)
+            }
+        } else {
+            setGroupedOptions([
+                {
+                    label: '',
+                    options: environments,
+                },
+            ])
+
+            if (!selectedOption) {
+                setSelectedOption(environments[0])
+            }
+        }
+    }, [environments])
+
+    const onChange = (selected: DeploymentChartOptionType) => {
+        setSelectedOption(selected)
+    }
+
+    return (
+        <ReactSelect
+            options={groupedOptions}
+            isMulti={false}
+            value={selectedOption}
+            classNamePrefix="compare-template-values-select"
+            formatOptionLabel={formatOptionLabel}
+            isOptionDisabled={(option) => option.value === 0}
+            isSearchable={false}
+            onChange={onChange}
+            components={{
+                IndicatorSeparator: null,
+                Option,
+                DropdownIndicator,
+                ValueContainer: customValueContainer,
+            }}
+            styles={{
```

Comment:
same here

Reply:
There're a few prop changes in these styles than others so keeping these as it is.